Meta/CMake/freedesktop/org.ladybird.Ladybird.metainfo.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>org.ladybird.Ladybird</id>
+
+  <name>Ladybird</name>
+  <summary>Truly independent web browser</summary>
+
```

Comment:
`<categories>` could be added here... the .desktop file has `Categories=Network;WebBrowser;Qt;KDE;`